START.md

```diff
@@ -202,6 +202,17 @@ You can use Taplytics to create [Experiments](https://taplytics.com/docs/android
 9. _That's it! Now build and run your app, you can start creating experiments with Taplytics!_
 
 ---
+### Install Using Segment
+1. Log in to your [Segment Account](https://segment.com?utm_source=partners&utm_medium=Docs&utm_campaign=taplytics.com) 
+2. Navigate to the integration page for the project you want to install Taplytics.
+3. Scroll through the integration list and turn on the integration for Taplytics.
+    ![Image of URL Scheme](http://taplytics.com/assets/docs/install-sdk/segment-integration.png)
+4. You will now be prompted to add your Taplytics API key. Log in to your [Taplytics Account](https://taplytics.com/login/) and navigate to settings. Once there copy your Taplytics iOS/Android SDK Key.
+    ![Image of URL Scheme](http://taplytics.com/assets/docs/install-sdk/taplytics-sdkkey.png)
+5. Head back to Segment and paste in your key. 
+6. Press Save and Enable in the right-hand corner. 
+
+---
 
```

Comment:
Would be interesting to keep it consistent across all the SDKs. Have a file called Segment.md and link to it from the START.md file?